Package does not work with the latest version of sequelize

I get the following error trying to install the package:

npm ERR! Found: sequelize@6.6.2
npm ERR! node_modules/sequelize
npm ERR!   sequelize@"^6.5.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer sequelize@"^5.0.0" from express-sequelize-crud@6.1.5

Having the same problem:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: **snip**@1.0.0
npm ERR! Found: sequelize@6.6.2
npm ERR! node_modules/sequelize
npm ERR!   sequelize@"^6.3.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer sequelize@"^5.0.0" from express-sequelize-crud@6.1.5
npm ERR! node_modules/express-sequelize-crud
npm ERR!   express-sequelize-crud@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

--- a/README.md
+++ b/README.md
@@ -83,12 +83,12 @@ import { User } from './models'
 const app = new express()
 app.use(
   crud('/admin/users', {
-    getList: ({ filter, limit, offset, order }) =>
+    getList: ({ filter, limit, offset, order, opts: { req, res } }) =>
       User.findAndCountAll({ limit, offset, order, where: filter }),
-    getOne: id => User.findByPk(id),
-    create: body => User.create(body),
-    update: (id, body) => User.update(body, { where: { id } }),
-    destroy: id => User.destroy({ where: { id } }),
+    getOne: (id, { req, res }) => User.findByPk(id),
+    create: (body, { req, res }) => User.create(body),
+    update: (id, body, {req, res}) => User.update(body, { where: { id } }),
+    destroy: (id, { req, res }) => User.destroy({ where: { id } }),
   })
 )
 ```

--- a/README.md
+++ b/README.md
@@ -120,7 +120,10 @@ app.use(
 )
 ```
 
-express-sequelize-crud, exposes a default search helper that you can use like this:
+express-sequelize-crud exposes a default search helper function `sequelizeSearchFields`. 
+Internally it uses `LIKE` syntax for all columns except uuid (`type: DataTypes.UUID`)
+
+Here is an example:
 
 ```ts
 import crud, {

